tests/baselines/reference/typeGuardOfFormTypeOfOther.types

```diff
@@ -63,7 +63,7 @@ else {
     var r2: string | C = strOrC; // string | C

```

Comment:
Correct. The behavior of negative clauses has been unified to simply be the negation of the positive clause, rather than an awkward second-set-of-special-cases-making-things-fail-in-else-blocks. I do need to go through and update all the comments.